atlasdb-ete-tests/src/main/java/com/palantir/atlasdb/AtlasDbEteServer.java

```diff
@@ -97,12 +101,14 @@ private TransactionManager createTransactionManagerWithRetry(AtlasDbConfig confi
         throw new IllegalStateException("Timed-out because we were unable to create transaction manager");
     }
 
-    private TransactionManager createTransactionManager(AtlasDbConfig config, Environment environment) {
+    private TransactionManager createTransactionManager(AtlasDbConfig config, Environment environment,
+            Optional<AtlasDbRuntimeConfig> atlasDbRuntimeConfigOptional) {
         return TransactionManagers.builder()
                 .config(config)
                 .schemas(ETE_SCHEMAS)
                 .registrar(environment.jersey()::register)
                 .userAgent("ete test")
+                .runtimeConfigSupplier(() -> atlasDbRuntimeConfigOptional)
```

Comment:
OK, this works. I think we probably want a comment somewhere that the runtime configuration is not actually live reloaded, in case we try to use this for live reloading.

Reply:
ah, i forgot about this. this is probably a requirement before shipping, since it needs to be able to be turned off live

Reply:
actually i think i'm referring to something different - the QosClient does not respect the live-reloadability of the QoS config

Reply:
yeah. There are 2 things here I guess -
1. the rate limiters take a config directly, so don't support live reload
2. the atlas ete server gives a constant supplier, so it doesn't live reload either

Reply:
The `AtlasDbEteServer` does not need the live reloading as of now unless we want to test the live-reloadability here.

Reply:
@jeremyk-91 @nziebart Just checking: are these comments now resolved elsewhere?

Reply:
As Himangi mentions we aren't dealing with 2 for now.

1 is done - see `QosRateLimiter` https://github.com/palantir/atlasdb/blob/feature/qos-service-api/qos-service-impl/src/main/java/com/palantir/atlasdb/qos/ratelimit/QosRateLimiter.java.

Aside: the concurrency in that class feels a bit weird (I think it is correct, but could result in multiple creations on a change - though last-one-wins is probably fine), but that's not the scope of this.